_posts/2015-06-26-ux-part-2.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post
+title: The OME UX Team. Part 2 - The OME-UX Development Workflow
+categories: user-experience
+---
+*By Gus Ferguson*
+
+In this post we will run through a real workflow that has been done over the last few releases of OMERO.
+
+The diagram below outlines our day-to-day pragmatic approach to developing.
+
+<img src="/images/workflow.png" alt="A diagram of the workflow of OME User Centred Design" style="width: 600px;"/>
+
+*Diagram of workflow of User Centred Design.*
+
+##Requirements Gathering
+
+Most of the UX team work starts with requirements in some form or another. This can range from a formal request through our forums or mailing lists to informal requests from users at meetings or even people sidling up to Jason in the pub and telling him they need OMERO to do the next big thing.
+
+The requirement can often be a single use-case from a bench scientist, ranging from a simple task through to a large, complex workflow. This can be entirely new or a request for enhancement (RFE) of an existing workflow or piece of functionality. Commonly, the requirement is specific to a relatively small number of users, and they are often very busy people and may be located in many different parts of the world.
+
+This means it can be a challenge spending enough time with a wide enough range of users to ensure the requirements are not too narrow and generalise to be applicable to the widest range of users and use cases as possible. However we generally find that once people are on board they are very generous with their time and expertise.
```

Comment:
```the requirements are not too narrow and generalise```
something like that will be easier to get:
```the requirements are not too narrow and generalise enough so that they are applicable to ....```